cli/flox/src/commands/environment.rs

```diff
@@ -1260,6 +1261,10 @@ impl Default for PullSelect {
 /// Pull environment from flox hub
 #[derive(Bpaf, Clone)]
 pub struct Pull {
+    /// forceably add current systems to the environment, even if incompatible
```

Comment:
Consistent capitalization would be nice:
```
    -a, --add-system  forceably add current systems to the environment, even if incompatible
    -d, --dir=<path>  Directory to create the environment in (default: current directory)
    -r, --remote=<owner/name>  ID of the environment to pull
    <owner/name>      ID of the environment to pull
    -d, --dir=<path>  Directory containing a managed environment to pull
    -f, --force       forceably overwrite the local copy of the environment
```

```suggestion
    /// Forceably add current system to the environment, even if incompatible
```